src/Commands/GenarateModel.js

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs';
+import shell from "shelljs";
+
+import "colors";
+
+export default async function (options, args) {
+  // remove env, pwd and first command
+  args = args.slice(3);
+  args = args.map(option => option.replace('--', ''));
+  const modelName = args[0][0].toUpperCase() + args[0].slice(1).toLowerCase();
+  const functions = {
+    table: `  get table() {\n    return \"${modelName}\";\n  }`,
```

Comment:
I think that we should store these string templates in string files, not in the code. We should load that templated from string files then. What do you think?